### DISCUSSION TITLE
Ride-hailing data from the Namma Yatri app for research puposes?

### DISCUSSION THREAD

**Discussion by ad93** [nammayatri/nammayatri#772] [Q&A] [answered]
Hi, 

I was wondering if we can obtain ride-hailing data that Namma Yatri uses for research and development purposes. If there is a provision then, can you let me know whom should I contact?

**Comment by sidsethujuspay** [comment]
Hey @ad93,
Can you specify what kind of data are you looking for?

**Comment by Prithivirajkota** [comment, marked as answer]
@ad93 NammaYatri ride data is openly available at [https://www.nammayatri.in/open/](https://www.nammayatri.in/open/). There is an option to download as well under each table and graphs.
